extensions/wikia/AdEngine/js/video/videoAdFactory.js

```diff
@@ -1,44 +1,45 @@
-/* global define */
+/*global define*/
 define('ext.wikia.adEngine.video.videoAdFactory', [
-	'ext.wikia.adEngine.domElementTweaker',
 	'ext.wikia.adEngine.video.googleIma',
 	'ext.wikia.adEngine.video.vastUrlBuilder',
-	'wikia.log'
-], function (DOMElementTweaker, googleIma, vastUrlBuilder, log) {
+	'wikia.log',
+	'wikia.window'
+], function (googleIma, vastUrlBuilder, log, win) {
 	'use strict';
-
 	var logGroup = 'ext.wikia.adEngine.video.videoAdFactory';
 
 	function init() {
 		return googleIma.init();
 	}
 
-	function create(width, height, adSlot, slotParams) {
-		var vastUrl = vastUrlBuilder.build(width / height, slotParams);
+	function create(width, height, adSlot, slotParams, vastUrl, preventImaReload) {
```

Comment:
What do you think about instead of introducing a new flag in `create` method separate `play` and `reload` functionality and do something meaningful with it?